zerver/views/upload.py

```diff
@@ -156,6 +165,34 @@ def serve_local_file_unauthed(request: HttpRequest, token: str, filename: str) -
     return serve_local(request, path_id, url_only=False)
 
 
+def serve_local_avatar_unauthed(request: HttpRequest, path: str) -> HttpResponseBase:
```

Comment:
Should we add a docstring explaining the role of this function? I think facts I'd want that to include are that because we want avatars to be able to served from inside email notifications, for example, we don't require authentication on accessing them, relying on their URL being generated using the AVATAR_SALT secret to protect their confidentiality.

Reply:
Added a docstring.